modules/jaeger-config-ingester.adoc

```diff
@@ -88,3 +81,28 @@ spec:
       es:
         server-urls: http://elasticsearch:9200
 ----
+
+== Configuring Ingester for autoscaling
+
+The Ingester can be configured to autoscale on demand. You configure autoscaling by setting the `autoscale` parameter to `true` and specifying a value for `.spec.ingester.maxReplicas` along with a reasonable value for the resources that you expect the Ingester's pod to consume. If you do not set a value for `.spec.ingester.maxReplicas` the Operator will set it to `100`.
+
+By default, when there is no value provided for `.spec.ingester.replicas`, the Jaeger Operator creates  a Horizontal Pod Autoscaler (HPA) configuration for the Ingester.  For more information about HPA, refer to the link:https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/[Kubernetes documentation].
+
+The following is an example autoscaling configuration, setting the Ingester’s limits as well as the maximum number of replicas:
+
+.Ingester autoscaling example
+[source,yaml]
+----
+apiVersion: jaegertracing.io/v1
+kind: Jaeger
+metadata:
+  name: simple-streaming
+spec:
+  strategy: streaming
+  ingester:
+    maxReplicas: 8
```

Comment:
Just out of curiosity: why 8 for the ingester, but 5 for the collector? Is this also so in the upstream docs?

Reply:
That's from the upstream docs.